subseas_flex_fcst/environment_linux.yml

```diff
@@ -0,0 +1,236 @@
+name: cptmr
```

Comment:
I don't know... currently the code is identical to the seasonal case, except for the S/L/target values that are hard coded and for file reading.
Once S/L/target are soft-coded, I'd expect them to work the same for both case, maybe with minimal input in the config file.
Data file reading changed because of new version of CPT. I would expect anyway at some point that Maprooms would be agnostic to data reading to some extent.
So... right now, the specificity of this environment is: i) cpt reading libraries and ii) distributions calculations.
Maybe it should be flex_fcst_mr?